GitVersion.yml

```diff
@@ -9,7 +9,3 @@ branches:
   release:
     increment: Patch
     tag: ''
-ignore:
```

Comment:
@bording @ramonsmits this will likely cause issues with the versioning?

Reply:
Just verified that removing those entries does not alter the version calculated on `master`, so it's safe to remove them.